index.rst

```diff
@@ -679,14 +679,24 @@ exists in the repository. But with a bypass function the Butler can not know
 what object is actually needed and for example some mappers return information
 derived from the `location.dataId` and do not actually need the object itself.
 
+Standardize Functions
+^^^^^^^^^^^^^^^^^^^^^
+
+Sometimes objects require further processing after the Butler has instantiated
+the object. To do this the mapper may declare a function with the object's
+dataset type name, preceded by `std_`, that is: `std_<datasetType>`. The
+signature must be `std_<datasetType>(self, item)`, where `item` is the object
+to be standardized. The function must return the standardized `item`.
```

Comment:
Could you also mention if it is OK to modify `item` in-place (I presume it should be fine but better have it stated explicitly). Is this method allowed to throw exceptions?